packages/gatsby-plugin-theme-ui/src/babel.ts

```diff
@@ -25,7 +32,7 @@ const plugin = (_: Babel, options: Options) => {
       } = state
 
       if (filename === inFile) {
-        const parsed = parse(`export default ${JSON.stringify(theme)};`, {
+        const parsed = parse(template(JSON.stringify(JSON.stringify(theme))), {
```

Comment:
Why JSON.stringify twice?

Reply:
If you `stringify` once, the case where there is a `"` inside a string gets badly scaped, breaking JSON.parse. Following this [random advice on stack overflow](https://stackoverflow.com/questions/5506000/json-stringify-doesnt-escape) fixed the issue

Reply:
I think a comment explaining that would be helpful